src/applications/gi/components/ResponsiveTable.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const borderClasses =
+  'vads-u-border-top--0 vads-u-border-right--0 vads-u-border-left--0 vads-u-padding--0 vads-u-padding-y--0p5 medium-screen:vads-u-padding--1';
+const rowPaddingClass = 'vads-u-padding-y--2';
+
+class ResponsiveTable extends React.Component {
+  renderHeader = field => {
+    return (
+      <th key={field} className={borderClasses} role="columnheader">
+        {field}
+      </th>
+    );
+  };
+
+  capitalizeFirstLetter = string => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
```

Comment:
This can be done with CSS

```css
td::first-letter {
  text-transform: capitalize;
}
```

Reply:
`src/applications/gi` doesn't need this but updated `src/applications/disability-benefits/view-payments` to do this